ios/BT/Storage/BTSecureStorage.swift

```diff
@@ -2,7 +2,7 @@ import ExposureNotification
 import Foundation
 import RealmSwift
 
-final class BTSecureStorage: SafePathsSecureStorage {
+class BTSecureStorage: SafePathsSecureStorage {
```

Comment:
why not `final`? 🤔  Are we subclassing `BTSecureStorage` anywhere?

Reply:
we are making BTSecureStorage not final so we can mock it, I can create a protocol and make this implementation final, what do you think?

Reply:
ah I see - thanks for explaining - either approach is fine 👍